edx_solutions_api_integration/mobile_api/tests/test_users.py

```diff
@@ -50,3 +60,170 @@ def test_user_with_orgs(self):
         response = self.api_response(data={'username': self.user.username})
         self.assertEqual(response.data['count'], 1)
         self.assertEqual(response.data['results'][0]['display_name'], org_name)
+
+
+@ddt.ddt
+class TestUserCourseGradesApi(MobileAPITestCase):
+    """
+    Tests for /api/server/mobile/v1/users/course/grades?username=<username>&course_id=<course_id>
```

Comment:
This URL seems wrong? Isn't it `/users/courses/grades` ?

Reply:
Thanks for catching this, will fix it.